modules/rosa-rhods.adoc

```diff
@@ -2,7 +2,7 @@
 //
 // * adding_service_cluster/rosa-available-services.adoc
 :_mod-docs-content-type: CONCEPT
-[id="rosa-rhods_{context}"]
-= Red{nbsp}Hat OpenShift Data Science
+[id="rosa-AI_{context}"]
+= Red{nbsp}Hat OpenShift AI
 
-{rhods} (RHODS) enables users to integrate data and AI and machine learning software to run end-to-end machine learning workflows. It provides a collection of notebook images with the tools and libraries required to develop and deploy data models. This allows data scientists to easily develop data models, integrate models into applications, and deploy applications using Red{nbsp}Hat OpenShift. RHODS is available as an add-on to Red{nbsp}Hat managed environments such as {osd} and {product-title} (ROSA).
+Red Hat OpenShift AI enables users to integrate data and AI and machine learning software to run end-to-end machine learning workflows. It provides a collection of notebook images with the tools and libraries required to develop and deploy data models. This allows data scientists to easily develop data models, integrate models into applications, and deploy applications using Red{nbsp}Hat OpenShift. OpenShift AI is available as an add-on to Red{nbsp}Hat managed environments such as OpenShift Dedicated and Red{nbsp}Hat OpenShift Service on AWS (ROSA).
```

Comment:
Out of scope for this PR, but we need to come up with a new attribute for OpenShift AI.